### DISCUSSION TITLE
MQTT -> Mosquitto -> Telegraf -> Influxdb v2 -> Grafana

### DISCUSSION THREAD

**Discussion by toglader** [jomjol/AI-on-the-edge-device#1844] [Q&A]
Hi,

I'm trying to set up a connection from AI on the edge device to influx db and visualize data with Grafana. However, I have been stuck on Telegraf. What should I set as data_format (and possible data_type) to get connection to influxdb working? String-data type works some how but it is string and not an numberic value and therefore visualisation isn't working. 

Does anyone made such configuration and be able to share configuration? telegraf configuration in first would be very helpful for me.

**Comment by caco3** [comment]
I only use the path MQTT -> Mosquitto -> Homeassistant -> InfluxDB2 - Grafana.

Homeassistant interpretiert dabei die `value` correkt als float.

I don't  know how it works with Telegraf.

**Comment by Slider0007** [comment]
Maybe this answer helps somehow: https://github.com/jomjol/AI-on-the-edge-device/issues/1800#issuecomment-1379948408
